.github/workflows/label.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       with:
         labels: maintenance
     - uses: actions-ecosystem/action-add-labels@v1
-      if: startsWith(github.event.pull_request.head.ref, 'junk')
+      if: startsWith(github.event.pull_request.head.ref, 'junk') || startsWith(github.event.pull_request.head.ref, 'dependabot') || startsWith(github.event.pull_request.head.ref, 'pre-commit-ci-update-config')
```

Comment:
I'm wondering if we should do this per user/app. For example ignoring the following users:

- `app/pre-commit-ci`
- `app/github-actions`
- `app/dependable`

Reply:
Yes. Let's!

Reply:
Seems that we can use `github.actor` for this.